sparkly/testing.py

```diff
@@ -96,9 +96,7 @@ class SparklyTest(TestCase):
     test_target = None
 
     @classmethod
-    def setUpClass(cls):
-        super(SparklyTest, cls).setUpClass()
-
+    def _init_session(cls):
```

Comment:
Unfortunately it's needed, otherwise anything else we set up for `SparklyTest` has to be replicated for `SparklyGlobalTest`. Currently that includes fixtures and `test_target` importing.